libs/vre/shared/app-resource-page/src/lib/resource-representation.component.ts

```diff
@@ -15,25 +15,23 @@ import { filter, map } from 'rxjs/operators';
 
 @Component({
   selector: 'app-resource-representation',
-  template: ` <div class="representation-container center" [ngSwitch]="representationToDisplay.fileValue.type">
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      [attachedProject]="attachedProject$ | async"
-      *ngSwitchCase="representationConstants.stillImage"
-      [resource]="resource.res" />
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      *ngSwitchCase="representationConstants.externalStillImage"
-      [attachedProject]="attachedProject$ | async"
-      [resource]="resource.res" />
-
+  template: ` <div class="representation-container center" [ngSwitch]="true">
+    <ng-container *ngIf="attachedProject$ | async as attachedProject">
```

Comment:
@irmastnt Is this not working like expected because the `[ngSwitch]` directive is used in this `div` above instead of `ng-container` here? In [docs](https://v17.angular.io/api/common/NgSwitch) examples I see the switch cases are used in direct child elements. Please try it out and let us know.